sdk/sdk-api/src/main/java/mil/emp3/api/Point.java

```diff
@@ -90,11 +92,22 @@ public IGeoIconStyle getIconStyle() {
      */
     @Override
     public void setIconURI(final String sURL) {
+        //check if valid web URL
         if(android.webkit.URLUtil.isValidUrl(sURL)) {
             ((IGeoPoint) this.getRenderable()).setIconURI(sURL);
-        } else {
-            throw new IllegalArgumentException("Invalid Input, " + sURL + " is not a valid URL");
+            return;
+        }
+        //check if valid file path and that the file exists
+        final String FileUrlPrefix = "file:"
```

Comment:
missing ";" at the end of the line...this constant should be put at the top within the class (not in the method, so it is only instantiated once)